src/gas-price/gas-price.ts

```diff
@@ -171,6 +172,7 @@ export const getAirseekerRecommendedGasPrice = async (
     scalingWindow,
     maxScalingMultiplier,
   } = gasSettings;
+  initializeGasStore(chainId, providerName); // TODO: This could be moved to a better place to avoid calling it multiple times (in different functions).
```

Comment:
It could probably just be removed, We'd never call this function without first having collected gas prices, which means that the `gasPriceStore` would already be initialized.

Reply:
Not really, when we first call this function we have no gas prices stored.